src/fsdocs-tool/BuildCommand.fs

```diff
@@ -335,6 +338,61 @@ type internal DocContent
                                        template,
                                        outputFileFullPath
                                    )))
+                      elif isPynb then
+                          printfn "  preparing %s --> %s" inputFileFullPath outputFileRelativeToRoot
+
+                          let evaluateNotebook ipynbFile =
+                              let psi =
+                                  ProcessStartInfo(
```

Comment:
This will fail when the user doesn't have `dotnet repl` available.
![image](https://github.com/fsprojects/FSharp.Formatting/assets/2621499/a3c6e071-9fed-4780-9f42-32c7ae7097ac)

I think we want to check if we can use this by running `dotnet repl --version` first.
And, in any case, we probably want to wrap running external processes in a `try/with` block.